openapi.yaml

```diff
@@ -1778,18 +1778,18 @@ paths:
       tags:
         - taxes
       summary: Create a tax
-      description: Create a tax
+      description: This endpoint creates a new tax representing a customizable tax rate applicable to either the organization or a specific customer.
```

Comment:
Do we already put that it can be defined on the plan or a specific charge?

Reply:
I guess we will update it later